chef/cookbooks/provisioner/recipes/update_nodes.rb

```diff
@@ -341,7 +341,8 @@ def find_node_boot_mac_addresses(node, admin_data_net)
                       architecture: arch,
                       is_ses: storage_available && !cloud_available,
                       crowbar_join: "#{os_url}/crowbar_join.sh",
-                      default_fs: mnode[:crowbar_wall][:default_fs] || "ext4")
+                      default_fs: mnode[:crowbar_wall][:default_fs] || "ext4",
+                      needs_openvswitch: mnode[:network] && mnode[:network][:needs_openvswitch] || false)
```

Comment:
Can you add some `()` here, it's always a bit ambiguous to have `&&` and `||` in the same expression.